src/meltano/core/tracking/tracker.py

```diff
@@ -441,9 +441,25 @@ def track_block_event(self, block_type: str, event: BlockEvents) -> None:
             )
         )
 
+    def setup_exit_event(self):
+        from meltano import cli
```

Comment:
Can we get a doc string for this method? The line comments would make fine content for it.